src/ImageSharp/Processing/Processors/Quantization/PaletteQuantizer{TPixel}.cs

```diff
@@ -17,25 +17,31 @@ internal struct PaletteQuantizer<TPixel> : IQuantizer<TPixel>
         where TPixel : unmanaged, IPixel<TPixel>
     {
         private readonly EuclideanPixelMap<TPixel> pixelMap;
+        private readonly bool leaveMap;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PaletteQuantizer{TPixel}"/> struct.
         /// </summary>
         /// <param name="configuration">The configuration which allows altering default behaviour or extending the library.</param>
         /// <param name="options">The quantizer options defining quantization rules.</param>
         /// <param name="pixelMap">The pixel map for looking up color matches from a predefined palette.</param>
+        /// <param name="leaveMap">
+        /// <see langword="true"/> to leave the pixel map undisposed after disposing the <see cref="PaletteQuantizer{TPixel}"/> object; otherwise, <see langword="false"/>.
+        /// </param>
         [MethodImpl(InliningOptions.ShortMethod)]
         public PaletteQuantizer(
             Configuration configuration,
             QuantizerOptions options,
-            EuclideanPixelMap<TPixel> pixelMap)
+            EuclideanPixelMap<TPixel> pixelMap,
+            bool leaveMap)
```

Comment:
Nit: typical pattern I've seen for this is to have a bool parameter indicating ownership (not the lack of it), (-> `ownsMap`)

Feel free to ignore in this PR.

Reply:
I copied `DeflateStream`. That uses a `leaveOpen` parameter for the same thing.